### ISSUE TITLE
RS1025 false positive for overridden initialize method

### ISSUE THREAD

**Issue by sharwell** [dotnet/roslyn-analyzers#4624]
RS1025 should not report a diagnostic on the following code:

```csharp
public abstract class BaseAnalyzer : DiagnosticAnalyzer
{
    public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => throw null;

    public override void Initialize(AnalysisContext context)
    {
        context.EnableConcurrentExecution();
        context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
    }
}

public class DerivedAnalyzer : BaseAnalyzer
{
    public override void Initialize(AnalysisContext context)
    {
        base.Initialize(context);
    }
}
```

**Comment by mattleibow** [comment]
Duplicate of https://github.com/dotnet/roslyn-analyzers/issues/4625

**Comment by sharwell** [comment]
📝 This is not a duplicate (it affects a different diagnostic ID)
